src/pages/bubblegum/guides/javascript/how-to-create-1000000-nfts-on-solana.md

```diff
@@ -0,0 +1,695 @@
+---
+title: Create 1 Million NFTs on Solana
+metaTitle: Create 1 Million NFTs on Solana | Bubblegum
+description: How to Create a Compressed NFT Collection of 1 Million cNFTs on Solana using the Metaplex Bubblegum program.
+---
+
+If you are looking to launch an NFT Collection on Solana Solana the Sugar CLI toolkit will abstract some of the trickier setup and management steps for you providing you with a automated launch system to create Candy Machines on the Solana blockchain.
+
+## Prerequisite
+
+- Code Editor of your choice (recommended Visual Studio Code).
+- Node 18.x.x or above.
+- Basic knowledge of Javascript and running scripts.
+
+## Initial Setup
+
+This guide will run through create of an NFT Core Asset with Javascript based on a single file script. You may need to modify and move functions around to suit your needs.
+
+### Initializing
+
+Start by initializing a new project (optional) with the package manager of your choice (npm, yarn, pnpm, bun) and fill in required details when prompted.
+
+```js
+npm init
+```
+
+### Required Packages
+
+Install the required packages for this guide.
+
+{% packagesUsed packages=["umi", "umiDefaults", "bubblegum", "tokenMetadata", "@metaplex-foundation/umi-uploader-irys"] type="npm" /%}
+
+```js
+npm i @metaplex-foundation/umi
+```
+
+```js
+npm i @metaplex-foundation/umi-bundle-defaults
+```
+
+```js
+npm i @metaplex-foundation/bubblegum
+```
+
+```js
+npm i @metaplex-foundation/token-metadata
+```
+
+```js
+npm i @metaplex-foundation/umi-uploader-irys;
+```
```

Comment:
would it make it easier to have just one `npm i` command here? iirc you can concatenate the package names into one command.

Would result in less steps for the user. The list ove packages is above either way.

Reply:
I've never actually gotten this to work? Like ever. I've always tried using the multi line installs but they always error for me. Please show me the way ha!